flocx_market/db/sqlalchemy/bid_api.py

```diff
@@ -0,0 +1,38 @@
+from flocx_market.db.orm import orm
+from flocx_market.db.sqlalchemy.bid_model import BidModel
+from uuid import uuid4
+
+
+class BidApid(BidModel):
```

Comment:
Why is this `BidApid` and not just `BidApi`?

Reply:
mistake, will fix it